Source/Details/ASTraitCollection.m

```diff
@@ -92,7 +100,7 @@ ASPrimitiveTraitCollection ASPrimitiveTraitCollectionMakeDefault() {
     .displayGamut = UIDisplayGamutUnspecified,
     .userInterfaceIdiom = UIUserInterfaceIdiomUnspecified,
     .layoutDirection = UITraitEnvironmentLayoutDirectionUnspecified,
-    .preferredContentSizeCategory = ASPrimitiveContentSizeCategoryMake(AS_UIContentSizeCategoryUnspecified()),
+    .preferredContentSizeCategory = AS_UIContentSizeCategoryUnspecified(),
```

Comment:
👍

Reply:
Why do we remove the `ASPrimitiveContentSizeCategoryMake` in here. If you read the comment above `ASPrimitiveContentSizeCategory` we have to use `ASPrimitiveContentSizeCategoryMake`